staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2105,9 +2105,9 @@ type PersistentVolumeClaimTemplate struct {
 
 // ContainerPort represents a network port in a single container.
 type ContainerPort struct {
-	// If specified, this must be an IANA_SVC_NAME and unique within the pod. Each
-	// named port in a pod must have a unique name. Name for the port that can be
-	// referred to by services.
+	// If specified, this must be an IANA_SVC_NAME. Each named port in a pod
```

Comment:
Should we acknowledge that this is a compat thing?  E.g.

"""
If specified, this must be an IANA_SVC_NAME.  Each named port in a container must have a unique name.  Due to backwards compatibility, ports on different containers in a pod may have the same name, in which case Probes and Services which refer to ports by name will use the first instance of that name.  Intentional use of this behavior is discouraged.
"""